src/main/java/service/Parser.java

```diff
@@ -0,0 +1,8 @@
+package service;
+
+import java.util.List;
+import model.FruitTransaction;
+
+public interface Parser {
```

Comment:
```suggestion
public interface FruitTransactionParserService {
```
Don`t forgot anded word "Service" to name of your class
likewise in the following cases